change-validation/src/main/scala/bootstrap/rudder/plugin/ChangeValidationConf.scala

```diff
@@ -93,19 +93,22 @@ import com.normation.rudder.services.workflows.NodeGroupChangeRequest
 import com.normation.rudder.services.workflows.RuleChangeRequest
 import com.normation.rudder.services.workflows.WorkflowLevelService
 import com.normation.rudder.services.workflows.WorkflowService
-import java.nio.file.Paths
 import net.liftweb.common.Box
+import net.liftweb.common.EmptyBox
 import net.liftweb.common.Full
 
+import java.nio.file.Paths
+
 /*
  * The validation workflow level
  */
 class ChangeValidationWorkflowLevelService(
-    status:                    PluginStatus,
-    defaultWorkflowService:    WorkflowService,
-    validationWorkflowService: TwoValidationStepsWorkflowServiceImpl,
-    validationNeeded:          Seq[ValidationNeeded],
-    workflowEnabledByUser:     () => Box[Boolean]
+    status                   : PluginStatus
+  , defaultWorkflowService   : WorkflowService
+  , validationWorkflowService: TwoValidationStepsWorkflowServiceImpl
+  , validationNeeded         : Seq[ValidationNeeded]
+  , workflowEnabledByUser    : () => Box[Boolean]
+  , validatedUserRepo: RoValidatedUserRepository
 ) extends WorkflowLevelService {
```

Comment:
you must not use that syntax anymore, it is not auto-formater friendly. Please keep the other one